models/marts/core/fct_ga4__pages.sql

```diff
@@ -0,0 +1,45 @@
+with page_view as (
+    select
+        event_date_dt,
+        extract( hour from (select  timestamp_micros(event_timestamp))) as hour,
+        page_location,  -- includes query string parameters not listed in query_parameter_exclusions variable
+        concat( cast(event_date_dt as string), cast(extract( hour from (select  timestamp_micros(event_timestamp))) as string), page_location ) as page_key,
+        page_title,  -- would like to move this to dim_ga4__pages but need to think how to handle page_title changing over time
+        count(event_name) as visits,
```

Comment:
@dgitis ok to rename to "pageviews"? Visits is often associated with sessions. But this is a count of pageviews ,yes?

Reply:
I used page_views as it seems more consistent with other column names like new_users.